js/src/components/modals/PostSettingsModal.js

```diff
@@ -2,6 +2,107 @@ import { Button } from "@yoast/components/src/button/Button";
 import { __, sprintf } from "@wordpress/i18n";
 import { useState, useCallback } from "@wordpress/element";
 import Modal from "./Modal";
+import Collapsible from "../SidebarCollapsible";
+import SnippetEditor from "../../containers/SnippetEditor";
+import FacebookContainer from "../../containers/FacebookEditor";
+import TwitterContainer from "../../containers/TwitterEditor";
+import SchemaTabContainer from "../../containers/SchemaTab";
+import AdvancedSettings from "../AdvancedSettings";
+import PropTypes from "prop-types";
+
+/**
+ * Returns the ModalContent.
+ *
+ * We need this in a function since the scripData is not available earlier.
```

Comment:
```suggestion
 * We need this in a function since the scriptData is not available earlier.
```

Reply:
`scriptData` will be removed from this function and we will use the store to fetch the settings.